apps/renderer/src/index.css

```
@@ -218,6 +218,16 @@ body {
   overflow-x: auto;
 }
 
+/*dark mode colors for markmap svg's */
```

Stylelint (17.13.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)